package.json

```diff
@@ -53,7 +53,7 @@
     "duplexify": "^3.5.0",
     "express": "^4.17.1",
     "express-healthcheck": "^0.1.0",
-    "express-partial-templates": "^0.2.0",
+    "express-partial-templates": "^0.2.1",
```

Comment:
Are these 2 packages updates done because you know it is saved to do after doing the upgrade in file-vault? As file vault is usually running as a separate instance

Reply:
There were vulnerabilities in these packages which were flagged up in the file-vault pipeline